src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
 }
 
 @main(s"Opportunity ${opportunity.id.id}") {
-    <div class="grid-row">
-        <a href="@controllers.routes.OpportunityController.showOpportunities()">&lt; Return to opportunity list</a>
-        <p></p>
-        <h1>@opportunity.title</h1>
+<div class="grid-row">
 
+    <div class="column-full">
+        <p>
+            <a href="@controllers.routes.OpportunityController.showOpportunities()">&lt; Return to opportunity list</a>
+        </p>
+    </div>
+
+    <div class="column-two-thirds">
         <header class="page-header">
-            <table>
-                <tbody>
-                @titleCols.map { case (col1, col2) =>
-                <tr>
-                    <td>@Html(sectionLink(col1._1, col1._2))</td>
-                    <td>@col2.map { t => @Html(sectionLink(t._1, t._2)) }.getOrElse("")
-                </td>
-                </tr>
-                }
-                </tbody>
-            </table>
+            <h1 class="heading-xlarge">@opportunity.title</h1>
         </header>
 
+        <aside>
```

Comment:
Stying is diffent on axshare (black border, grey background) - deliberate departure from the design?